assets/js/modules/adsense/components/settings/SettingsForm.js

```diff
@@ -31,11 +31,17 @@ import {
 	ErrorNotice,
 	UseSnippetSwitch,
 } from '../common';
+import ProgressBar from '../../../../components/progress-bar';
 const { useSelect } = Data;
 
 export default function SettingsForm() {
 	const clientID = useSelect( ( select ) => select( STORE_NAME ).getClientID() );
 	const existingTag = useSelect( ( select ) => select( STORE_NAME ).getExistingTag() );
+	const hasResolvedGetExistingTag = useSelect( ( select ) => select( STORE_NAME ).hasFinishedResolution( 'getExistingTag' ) );
```

Comment:
Nitpick, but it would probably make sense to combine these like you did above (since they belong to the same selector call in a way).

Reply:
Updated.